requirements.txt

```diff
@@ -38,7 +38,7 @@ gunicorn==19.9.0
 rq==1.1.0
 rq-scheduler==0.9.1
 celery==4.3.0
-kombu==4.6.5
+kombu==4.6.6
```

Comment:
If I remember right `4.6.6` had a bug. As this change is probably not relevant for this PR, let's keep as is.

Reply:
Sounds good,  i bumped it cause of a bug we where seeing in the admin status view of celery and bumping the version resolved the issue, but i agree this should be part of a different PR

Reply:
Btw, `4.6.5` has a bug. `4.6.3` is the version I had in mind. We accidentally upgraded as part of the Python 3 migration and dependencies update... It's possible that `4.6.6` is good too, but as we deprecating Celery anyway we moved back to the last version that was stable for us.